client/src/buttons.ts

```diff
@@ -1,6 +1,15 @@
 import { ExtensionContext } from 'vscode';
 import * as vscode from 'vscode';
 
+const REACH_COMPILE_S:string = 'Reach Compile';
+const REACH_COMPILE:string = 'reach.compile';
```

Comment:
nit: Can you change the command constants to use a suffix of `_CMD`. 
e.g. `REACH_COMPILE => REACH_COMPILE_CMD`